utils/common/src/main/kotlin/ProcessCapture.kt

```diff
@@ -58,8 +58,8 @@ class ProcessCapture(vararg command: String, workingDir: File? = null, environme
         }
     }
 
-    private val tempDir = createTempDirectory("$command-process").toFile().apply { deleteOnExit() }
     private val tempPrefix = command.first().substringAfterLast(File.separatorChar)
+    private val tempDir = createTempDirectory("$tempPrefix-process").toFile().apply { deleteOnExit() }
```

Comment:
There is also `createOrtTempDir` which has more usages than `createTempDir`. 
Have you considered using that one? (I'm not 100% sure if `createOrtTempDir()` should actually be used, so maybe get a second approval)

Reply:
Pinging @sschuberth for advice.

Reply:
As `common-utils` must not depend on `ort-utils` we cannot use `createOrtTempDir()` here.

However, one might argue that we don't need / want `tempPrefix` in both the directory *and* the file names.

Maybe do

    private val tempDir = createTempDirectory(javaClass.simpleName).toFile().apply { deleteOnExit() }

and while at it, in another commit rename `tempPrefix` to `commandName` so that it all reads like

    private val tempDir = createTempDirectory(javaClass.simpleName).toFile().apply { deleteOnExit() }
    private val commandName = command.first().substringAfterLast(File.separatorChar)
    private val stdoutFile = tempDir.resolve("$commandName.stdout").apply { deleteOnExit() }
    private val stderrFile = tempDir.resolve("$commandName.stderr").apply { deleteOnExit() }

Reply:
Will `javaClass.simpleName` not always return `ProcessCapture` ?
Is is not better to have `commandName` as part of the temp file ?

Reply:
> Will `javaClass.simpleName` not always return `ProcessCapture` ?

Yes.

> Is is not better to have commandName as part of the temp file ?

`commandName` is still used for the names of the *files*, but not anymore for the name of the parent *directory*, which is fine, IMO.